src/main/java/net/dv8tion/jda/api/hooks/ListenerAdapter.java

```diff
@@ -279,6 +283,7 @@ public void onGenericRole(@Nonnull GenericRoleEvent event) {}
     public void onGenericRoleUpdate(@Nonnull GenericRoleUpdateEvent event) {}
     public void onGenericEmote(@Nonnull GenericEmoteEvent event) {}
     public void onGenericEmoteUpdate(@Nonnull GenericEmoteUpdateEvent event) {}
+    public void onGenericGuildMemberUpdateEvent(@Nonnull GenericGuildMemberUpdateEvent event) {}
```

Comment:
Move this to be just below `onGenericGuildMember`

Reply:
Also, remove `Event` from the method name

Reply:
Done